scripts/integration-test.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: integration-test
+  labels:
+    app: integration-test
+spec:
+  nodeName: windows_node # to be replaced by sed
+  securityContext:
+    windowsOptions:
+      hostProcess: true
+      runAsUserName: "NT AUTHORITY\\SYSTEM"
```

Comment:
let's experiment here, could you try the user with the lowest privilege first?

Reply:
In my previous experiments, anything less than SYSTEM hasn't worked. Let me try again locally and get back to you though.

Reply:
Yeah, even setting the username to `"NT AUTHORITY\\Local service"` causes commands like 
```
C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe /c & { $global:ProgressPreference = 'SilentlyContinue'; (Get-VHD -Path C:\\var\\lib\\kubelet\\plugins\\testplugin-693308.csi.io\\disk-693308.vhdx).DiskNumber }
```
to fail with permission issues.

Reply:
And Network service fails too